src/apiserver/service/serviceconfig.go

```diff
@@ -123,17 +123,17 @@ func DeleteServiceByID(serviceID int64) (int64, error) {
 	return num, nil
 }
 
-func GetServiceList(name string, userID int64, source *int, sourceid *int64) ([]*model.ServiceStatusMO, error) {
-	query := model.ServiceStatusFilter{Name: name, Source: source, SourceID: sourceid}
+func GetServiceList(name string, projectID int64, userID int64, source *int, sourceid *int64) ([]*model.ServiceStatusMO, error) {
+	query := model.ServiceStatusFilter{Name: name, ProjectID: projectID, Source: source, SourceID: sourceid}
```

Comment:
**I think we should change or add a SQL query instead to pass a specific parameter.**

Reply:
This change is reuse the SQL，and improve the function of "ServiceStatusFilter"。